make.js

```diff
@@ -140,7 +141,8 @@ target.web = function() {
         ],
         preprocess: [
             [SRC_DIR + 'crxviewer.html', dest_dir],
-            [SRC_DIR + 'crxviewer.js', dest_dir]
+            [SRC_DIR + 'crxviewer.js', dest_dir],
+            [SRC_DIR + 'lib/crx-to-zip.js', dest_dir + 'lib/crx-to-zip.js']
```

Comment:
Please add a comment after this line, and remove `/crx-to-zip.js` at the right (also above).

After that we're good to go.